### PR TITLE
Fix: Revert to default block explorer button wasn't working

### PR DESCRIPTION
**Pull request by NicolasDorier**
Fix button that wasn't working

image

### CODE REVIEW

File: BTCPayServer/Views/UIServer/Policies.cshtml [hunk 1]
```diff
@@ -200,8 +200,8 @@
                     <label asp-for="BlockExplorerLinks[i].Link" class="form-label">@network.DisplayName (@network.CryptoCode)</label>
                     <input type="hidden" asp-for="BlockExplorerLinks[i].CryptoCode" value="@network.CryptoCode"/>
                     <div class="input-group">
-                        <input type="text" class="form-control" asp-for="BlockExplorerLinks[i].Link" value="@linkValue" rel="noreferrer noopener" data-default-link="@network.BlockExplorerLinkDefault"/>
-                        <button type="button" class="btn btn-secondary only-for-js" title="Revert to default">
+                        <input type="text" class="form-control" asp-for="BlockExplorerLinks[i].Link" value="@linkValue" rel="noreferrer noopener" />
```
Comment (Kukks) [inline BTCPayServer/Views/UIServer/Policies.cshtml:203]:
```suggestion
                        <input type="text" class="form-control block-explorer-link" asp-for="BlockExplorerLinks[i].Link" value="@linkValue" rel="noreferrer noopener" />
```

File: BTCPayServer/Views/UIServer/Policies.cshtml [hunk 2]
```diff
@@ -215,7 +215,8 @@
     <script>
         for (let element of document.getElementsByClassName("revert-default")) {
             element.addEventListener("click", function (){
-              element.parentElement.previousElementSibling.value = element.parentElement.previousElementSibling.getAttribute("data-default-link")
+            const defaultLink = element.getAttribute("data-default-link");
+            element.parentElement.querySelector('input').value = defaultLink;
```
Comment (Kukks) [inline BTCPayServer/Views/UIServer/Policies.cshtml:219]:
```suggestion
            element.parentElement.querySelector('input.block-explorer-link').value = defaultLink;
```

**Review by Kukks — CHANGES_REQUESTED** [review]
made it a bit more sturdy

**Review by dennisreimann — APPROVED** [review]
grafik

**Pull request merged**